extensions/coap/server/CoapServer.h

```diff
@@ -183,7 +183,7 @@ class CoapServer : public core::Connectable {
   /**
    * Determines if we are connected and operating
    */
-  virtual bool isRunning() {
+  virtual bool isRunning() const {
```

Comment:
I think this should be `override` instead of `virtual`

Reply:
Updated in a20c3a130d80bd73e4383b8161e23c52c77fb1fd